src/main/java/com/metsci/laproc/tools/DataSheetPanel.java

```diff
@@ -48,11 +56,17 @@ public void setDataSheet(ClassifierDataSet data) {
 		if (defaults.get("Table.alternateRowColor") == null)
 			defaults.put("Table.alternateRowColor", new Color(240, 240, 240));
 
-		this.add(scrollPane);
+		this.panel.add(scrollPane);
 		JButton newEvalSetButton = new JButton("Create New Eval Set");
-        NewEvalSetActionListener nesaInstance = new NewEvalSetActionListener(this.window, tableDisplayer);
-		newEvalSetButton.addActionListener(nesaInstance);
-		this.add(newEvalSetButton);
+
+		ActionListener listener = new ActionListener() {
```

Comment:
This seems like an ActionListener that we might make enough to warrant its own class.